composables/useMember.js

```diff
@@ -36,16 +36,47 @@
  * @property {Object.<string, any>} open_source_contributions.wikimedia_contributions
  */
 
+const STORAGE_KEYS = {
+  userId: 'userId',
+  member: 'member'
+}
+
 /**
  * Returns current (authenticated) member data
  * @returns { Ref<MemberData> }
  */
 export const useMemberData = () => useState('member', () => { })
 
+/**
+ * returns the user id
+ * @returns {Ref<string | null>}
+ */
+export const userId = () => useState('userId', () => null)
+
+/**
+ * updates userId and sets its value in localStorage
+ * @param {string | null} id 
+ */
+export const updateUserId = (id) => {
+  userId().value = id
+  id ?
+    localStorage.setItem(STORAGE_KEYS.userId, id) :
+    localStorage.removeItem(STORAGE_KEYS.userId)
```

Comment:
Is there any check if the user changed the member ID from the local storage in the console?